cram_common/cram_common_failures/cram-common-failures-tests.asd

```diff
@@ -0,0 +1,56 @@
+;;; Copyright (c) 2012, Amar Fayaz <amar@uni-bremen.de>
+;;; All rights reserved.
+;;; 
+;;; Redistribution and use in source and binary forms, with or without
+;;; modification, are permitted provided that the following conditions are met:
+;;; 
+;;;     * Redistributions of source code must retain the above copyright
+;;;       notice, this list of conditions and the following disclaimer.
+;;;     * Redistributions in binary form must reproduce the above copyright
+;;;       notice, this list of conditions and the following disclaimer in the
+;;;       documentation and/or other materials provided with the distribution.
+;;;     * Neither the name of the Intelligent Autonomous Systems Group/
+;;;       Technische Universitaet Muenchen nor the names of its contributors 
+;;;       may be used to endorse or promote products derived from this software 
+;;;       without specific prior written permission.
+;;; 
+;;; THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+;;; AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+;;; IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+;;; ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE
+;;; LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+;;; CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+;;; SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+;;; INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+;;; CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+;;; ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+;;; POSSIBILITY OF SUCH DAMAGE.
+
+(defsystem cram-common-failures-tests
+  :author "Amar Fayaz"
+  :license "BSD"
+  
+  :depends-on (lisp-unit
+               roslisp
+               cram-language
+               cram-prolog
+               cl-urdf
+               cram-common-failures
+               cram-bullet-reasoning
+               cram-pr2-description
+               cram-robot-interfaces
+               cl-transforms
+               cl-transforms-stamped
+               cram-designators
+               cram-object-knowledge
+               cram-location-costmap
+               cram-btr-visibility-costmap
+               cram-btr-spatial-relations-costmap
+               cram-occupancy-grid-costmap
+               cram-robot-pose-gaussian-costmap)
```

Comment:
Are all these dependencies really necessary? I doubt that. E.g., cram-language or cram-prolog, I don't see any prolog code or any CPL code.

Reply:
both cpl and prolog are being used in [line 79](https://github.com/cram2/cram/pull/174/files#diff-22b1f4fa4877bdc39600856b196d41e3R79) and [line 48](https://github.com/cram2/cram/pull/174/files#diff-22b1f4fa4877bdc39600856b196d41e3R48) of the test code.

I kept the other costmaps so as it stays as it is when we use the robot, in case any of them adds a change. All the other packages are used as well.

Reply:
indeed, my bad